### PR TITLE
Test exporting objects data to file (snapshot only)

### LINKED ISSUE #394: [Test] import/export functions
We need tests for the import/export functions that test importing actual spreadsheets and json files.

### LINKED ISSUE #486: [frontend] Complete TODO Unit Tests
- **user-permission-tests**
- [x] An instructor can access instructor routes
- [x] A TA can access TA routes
- [x] An instructor who is also a TA can access instructor and TA routes

- **template-tests**
- [x] upload a template

- **reporting-tag-tests**
- [ ] get all reporting_tags associated to positions for session
- [ ] get all reporting_tags associated to wage_chunks for session
- [ ] get reporting_tags for position
- [ ] get reporting_tags for wage_chunk
- [ ] create and delete reporting_tags for position
- [ ] create and delete reporting_tags for wage_chunk

- **import-export-tests**
- [x] Export data to a JSON/CSV/XLSX
- [x] Import data from a JSON/CSV/XLSX

- **ddah-tests**
- [x] getting ddahs for one session will not return ddahs for another session

### PR DESCRIPTION
**Pull request by Altair59**
test exporting instructor, applicant, position, assignment, and ddah object data to JSON/CSV/XLSX file

### CODE REVIEW

File: frontend/src/tests/import-export.test.js [hunk 1]
```diff
@@ -21,7 +44,7 @@ describe("Import/export library functionality", () => {
         primaryKey: "utorid",
     };
 
-    const instructorData = [
+    const instructorData1 = [
```
Comment (Accelsnow) [inline frontend/src/tests/import-export.test.js]: what is the difference between instructorData1 defined here and the global instructorData?

Comment (Altair59) [reply frontend/src/tests/import-export.test.js]: thanks. removed the redundant instructorData1

File: frontend/src/tests/import-export.test.js [hunk 2]
```diff
@@ -1,8 +1,31 @@
...
+// BELOW ARE USED FOR ROUND TRIP TEST FOR prepareData
+// import { prepareData } from "../views/instructors/import-export";
+// import XLSX from "xlsx";
+
+// function File(fileBits, fileName, options) {
+//     this.fileBits = fileBits;
+//     this.fileName = fileName;
+//     this.options = options;
+// }
+// global.File = File;
```
Comment (siefkenj) [inline frontend/src/tests/import-export.test.js]: Why are these comments here?

Comment (Altair59) [reply frontend/src/tests/import-export.test.js]: These are for creating a shim for File object which is only used in round trip test, which currently has some issues

**Comment by Altair59** [comment]
The round trip test is removed for this PR, but should be added after these refactorings:
1. re-organize all util functions in `importExportUtils` and `exportUtils` into separate files under` import-export` under libs
2. move all object's `prepareData` function into `import-export`

File: frontend/src/tests/import-export.test.js (hunk 2 above)
Comment (Altair59) [reply frontend/src/tests/import-export.test.js]: removed round trip test for this PR

File: frontend/src/tests/import-export.test.js [hunk 3]
```diff
@@ -107,6 +97,80 @@ describe("Import/export library functionality", () => {
...
+        expect(assignmentJson).toMatchSnapshot();
+    });
+
+    it("Export Ddahs to JSON/CSV/XLSX", () => {
+        // prepare ddah spreadsheet
+        const ddahSpreadsheet = prepareSpreadsheet.ddah(ddahData);
+        expect(ddahSpreadsheet).toMatchSnapshot();
+
+        // prepare ddah json
+        let ddahJson = [];
```
Comment (siefkenj) [inline frontend/src/tests/import-export.test.js]: Change these to use `ddahData.map`, etc. throughout.

**Pull request merged**